src/main/java/com/epam/controller/ToursController.java

```diff
@@ -104,7 +105,7 @@ public ModelAndView addTour(@RequestParam String tourType, @RequestParam String
                     .pricePerUnit(addPricePerPerson)
                     .hotelId(1) //stub
                     .description(tourDescription)
-                    .discountId(1) //stub
+                    .discountId(addDiscount)
```

Comment:
Is it still the name `discountId`? Maybe would better rename

Reply:
done